etc/apache2/includes/instance-oo.include.template

```diff
@@ -0,0 +1,85 @@
+
+    # Error pages
+    ErrorDocument 404 /404.html
+    ErrorDocument 500 /500.html
+    ErrorDocument 502 /502.html
+    ErrorDocument 503 /503.html
+
+    # don't loose time with IP address lookups
+    HostnameLookups Off
+
+    # needed for named virtual hosts
+    UseCanonicalName Off
+
+    # configures the footer on server-generated documents
+    ServerSignature Off
+
+    <Directory />
+        Options FollowSymLinks
+        AllowOverride None
+    </Directory>
+
+    DocumentRoot ${ADT_DATA}/var/www/
+    <Directory ${ADT_DATA}/var/www/>
+        Options Indexes FollowSymLinks MultiViews
+        AllowOverride None
+        Order allow,deny
+        allow from all
+    </Directory>
+
+    Alias /icons/ "/usr/share/apache2/icons/"
+    <Directory "/usr/share/apache2/icons">
+        Options Indexes MultiViews
+        AllowOverride None
+        Order allow,deny
+        Allow from all
+    </Directory>
+
+    Alias /logs/ "${DEPLOYMENT_DIR}/logs/"
+    <Directory "${DEPLOYMENT_DIR}/logs/">
+        Options Indexes MultiViews
+        AllowOverride None
+        Order allow,deny
+        Allow from all
+    </Directory>
+
+    ProxyRequests           Off
+    ProxyPreserveHost       On
+    ProxyPass               /404.html            !
+    ProxyPass               /500.html            !
+    ProxyPass               /502.html            !
+    ProxyPass               /503.html            !
+    ProxyPass               /robots.txt          !
+    ProxyPass               /style.css           !
+    ProxyPass               /logs/               !
+    ProxyPass               /icons/              !
+    ProxyPass               /images/favicon.ico  !
+    ProxyPass               /images/Arrow.gif    !
+    ProxyPass               /images/BgBlock.jpg  !
+    ProxyPass               /images/Header.png   !
+    ProxyPass               /images/Footer.png   !
+    ProxyPass               /images/Logo.png     !
+
+    ProxyPass               /elasticsearch       http://localhost:${DEPLOYMENT_ES_HTTP_PORT}
+    ProxyPassReverse        /elasticsearch       http://localhost:${DEPLOYMENT_ES_HTTP_PORT}
+
+	# Forward request to OnlyOffice when the url start with the version scheme (see in side the Docker container for it)
+	# ex: /v5.2.8-24/*
+	ProxyPassMatch    "^\/(v[\d]+\.[\d]+\.[\d]+[\.|-][\d]+\/.*)$"    http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
+	ProxyPassMatch	  "^\/OfficeWeb\/.*)$"							 http://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1 acquire=1000 retry=30
+
+	# Add header for X-Forwarded-For / X-Forwarded-Host / X-Forwarded-Server
+	ProxyAddHeaders On
+	# Add header X-Forwarded-Proto to enforce https on the underlying webserver
+	RequestHeader set X-Forwarded-Proto "https"
+
+	# Proxify of websocket url pattern
+	ProxyPassMatch    (.*)(\/websocket)$                            ws://localhost:${DEPLOYMENT_ONLYOFFICE_HTTP_PORT}/$1$2
```

Comment:
you should specify the specific ws configuration

Reply:
fixed in all templates